jobs/postgres/templates/create-database.erb

```diff
@@ -34,4 +34,7 @@ for DBNAME in "${DBNAMES[@]}"; do
 
   echo "Trying to alter user (to change password)..."
   $PACKAGE_DIR/bin/psql -d $DBNAME -h $HOST  -p $PORT -U vcap -c "alter role \"$USER\" with password '$PASSWORD'"
+
+  echo "Grant permission to user"
+  $PACKAGE_DIR/bin/psql -d $DBNAME -h $HOST  -p $PORT -U vcap -c  "grant all on schema public to \"$USER\";"
```

Comment:
I was wandering why we need this now?

Reply:
I don't know if some permissions changed on postgres 13 -> 15 but the user wasnt able to migrate db for BOSH on new instances.